components/shared/NavBarClient.tsx

```diff
@@ -27,44 +29,105 @@ export default function NavBarClient({ results }: NavBarClientProps) {
   }, []);
 
   const { navigationBar } = results || {};
-  const navItems = navigationBar?.leftNavItem;
+  const leftNavItems = navigationBar?.leftNavItem;
+  const rightNavItems = navigationBar?.rightNavItem;
   const logo = navigationBar?.Logo;
 
+  
 
-  console.log(navItems);
-  const navbarHeight = 120; 
+  const navbarHeight = 120;
```

Comment:
Is there a reason this one isn't regular tailwind?

Reply:
moved into tailwind :)